examples/wms.ipynb

```diff
@@ -185,10 +173,10 @@
     "    ge.QueryRectangle(\n",
     "        [-180.0, -90.0, 180.0, 90.0],\n",
     "        [time, time],\n",
-    "        resolution=(1.8, 1.8)\n",
-    "        ),\n",
-    "    colorizer_min_max=(0, 255)\n",
-    ")"
+    "        resolution=(0.8, 0.8)\n",
+    "        ), \n",
+    "        ge.workflow.Colorizer((0, 254), \"RdBu\", 25)\n",
```

Comment:
Could you specify a nicer-looking colorizer for the example notebook? We only used the grayscale since we had no better option yet :smile:.